include/DataFrame/DataFrame.h

```diff
@@ -89,7 +89,13 @@ class LIBRARY_API DataFrame : public ThreadGranularity {
     DataFrame &operator= (const DataFrame &) = default;
     DataFrame &operator= (DataFrame &&) = default;
 
-    ~DataFrame() = default;
+    ~DataFrame() {
+        column_list_.clear();
+        column_tb_.clear();
+        indices_.clear();
+        const SpinGuard guard(lock_);
+        data_.clear();
```

Comment:
I don't think you need the clears, except for 
```
        const SpinGuard guard(lock_);
        data_.clear();
```

And thanks for fixing this

Reply:
Yes, you're right. I just wanted to keep the order as the default destructor would have, but this works too of course.